cumulusci/core/keychain/BaseProjectKeychain.py

```diff
@@ -142,6 +142,11 @@ def _remove_org(self, name, global_org):
         del self.orgs[name]
         self._load_orgs()
 
+    # TODO: Are these used in any context other than
+    # encrypted_file_project_keychain?
+    #
+    # If not, they can be removed after a reasonable
+    # period of backwards compatibility
```

Comment:
Not sure I understand the comment. set_org is the standard interface for saving org configs to a keychain, so yeah, this is used by any keychain that doesn't override it.

Reply:
Sorry the comment should be about 10 lines lower around get_default, set_default, unset_default.

Do we use default orgs in contexts other than the command line/encrypted project?

Basically I'm not thrilled about having two overlapping ways of defining the default org, using the filesystem in some places and using a flag in others. If we don't need the flag, we should get rid of it.

Reply:
The web app keychains don't use default orgs, afaik. It's conceivable that someone using the CLI could use the EnvironmentProjectKeychain, which inherits these from BaseProjectKeychain. But setting a default org when using that keychain wouldn't actually work since it doesn't get persisted anywhere. So it would probably be good to raise a clear error about that anyway.